scripts/imagebuild.sh

```diff
@@ -25,11 +25,11 @@ dockerfile_dir="${base_dir}/images/federation-v2"
 
 echo "travis tag: ${TRAVIS_TAG}"
 echo "travis branch:${TRAVIS_BRANCH}"
-if [[ "${TRAVIS_TAG}" =~ ^[0-9]+\.[0-9]+\.[0-9]?\d+([-a-zA-Z0-9-]*)? ]]; then
-    echo "Pushing images with tag '${TRAVIS_TAG}'."
+if [[ "${TRAVIS_TAG}" =~ ^v([0-9]\.)+([0-9])[-a-zA-Z0-9]*([.0-9])* ]]; then
+   echo "Using tag: '${TRAVIS_TAG}'."
```

Comment:
Might want to say `Using image tag`, since there are a couple different types of tags this could refer to

Reply:
Yes you're right.